pdns/rec_channel_rec.cc

```diff
@@ -370,30 +370,39 @@ string doSetCarbonServer(T begin, T end)
 }
 
 template<typename T>
-string doSetDnssecLogBogus(T begin, T end)
+string doSetDnssecLog(T begin, T end)
 {
   if(checkDNSSECDisabled())
-    return "DNSSEC is disabled in the configuration, not changing the Bogus logging setting\n";
+    return "DNSSEC is disabled in the configuration, not changing the DNSSEC validation logging setting\n";
 
   if (begin == end)
-    return "No DNSSEC Bogus logging setting specified\n";
+    return "No DNSSEC validation logging setting specified\n";
 
   if (pdns_iequals(*begin, "on") || pdns_iequals(*begin, "yes")) {
-    if (!g_dnssecLogBogus) {
+    if (g_dnssecLogMode != DNSSECLogMode::On) {
       L<<Logger::Warning<<"Enabling DNSSEC Bogus logging, requested via control channel"<<endl;
-      g_dnssecLogBogus = true;
+      g_dnssecLogMode = DNSSECLogMode::On;
       return "DNSSEC Bogus logging enabled\n";
```

Comment:
The two messages above mention "Bogus" logging but it will enable the logging of all validation results.